test/e2e/downloads_test.go

```diff
@@ -0,0 +1,50 @@
+package e2e
+
+import (
+	"net/http"
+	"testing"
+
+	operatorsv1 "github.com/openshift/api/operator/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/console-operator/pkg/api"
+	"github.com/openshift/console-operator/pkg/console/controllers/clidownloads"
+	routesub "github.com/openshift/console-operator/pkg/console/subresource/route"
+	"github.com/openshift/console-operator/test/e2e/framework"
+)
+
+func setupDownloadsTestCase(t *testing.T) (*framework.ClientSet, *operatorsv1.Console) {
+	return framework.StandardSetup(t)
+}
+
+func cleanupDownloadsTestCase(t *testing.T, client *framework.ClientSet) {
+	framework.StandardCleanup(t, client)
+}
+
+func TestDownloadsEndpoint(t *testing.T) {
+	client, _ := setupDownloadsTestCase(t)
+	defer cleanupDownloadsTestCase(t, client)
+
+	route, err := client.Routes.Routes(api.OpenShiftConsoleNamespace).Get(api.OpenShiftConsoleName, v1.GetOptions{})
+	if err != nil {
+		t.Fatalf("error, can't get route: %s", err)
```

Comment:
nit: would use the same string as we use on most of the places where we are erroring out on GET
```suggestion
		t.Fatalf("could not get route: %s", err)
```

Reply:
thx, updated!